auth-web/src/components/auth/create-account/PremiumChooser.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+<v-container>
+  <v-form ref="premiumAccountChooser" lazy-validation>
+    <p class="mt-3">Do you want to link this account with an existing BC Online Account? <a href="">Learn more</a></p>
```

Comment:
there seems to be a modal popup for learn more in the uxpin.
not sure we are implementing that scenario later or now?

Reply:
@jeznorth will handle the modal popup